src/__tests__/orchestrator.integration.spec.ts

```diff
@@ -13,8 +13,9 @@ import { readFileSync } from 'fs'
 
 describe('elastic orchestrator', () => {
   describe('indices need updating', () => {
-    const indexConfigFile = 'index-00001.json'
-    const anotherConfigFile = 'index-00002.json'
+    const indexConfig1 = 'index-00001.json'
+    const indexConfig2 = 'index-00002.json'
+    const indexConfig3 = 'index-00003.json'
```

Comment:
Can we keep the names with the context please. Maybe `indexConfig3` needs to be called `newLocalConfig` or something. I don't often push for this but I dislike 'indexConfig1/2/3' as it's not telling me what that represent. 

I don't think you even need 2 or 3 at this level as they're not used by any other tests bar one are they so they could be named better inside the test that cares about them.

Reply:
Updated